docs/UserGuide.md

```diff
@@ -167,6 +168,10 @@ Format: `remove INDEX [p/[PHONE]] [p/[GITHUBPROFILE]] [e/[EMAIL]] [a/[ADDRESS]]
 * Clears the field value(s) when no keyword is provided for the field.
   * e.g. `t/` will remove all the tags associated with the contact.
 
+<div markdown="span" class="alert alert-primary"> :bulb: Tip
+`remove` command considers **removing a field successfully** when no keyword is provided even if the field is empty.
```

Comment:
Perhaps consider "If no keyword is provided, `remove` will still execute successfully on an empty field"

Just some concerns as describing this behavior is rather hard